client/pom.xml

```diff
@@ -137,6 +135,7 @@
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
+            <version>${junit.version}</version>
```

Comment:
Isn't this a managed dependency inherited from the root pom? Why add the version number explicitly?